src/lib/context.ts

```diff
@@ -1,9 +1,34 @@
 import { createContext } from "react";
+import { Plurals } from "../components/Ditto";
+
+interface StructuredTextData {
+  plurals: Plurals;
+  text: string;
+  variables: Variables;
+}
+type FlatTextData  = string
+
+export interface TextData  {
+  plurals: Plurals;
+  text: string;
+  variables: VariableData;
+}
 
 export type Count = number | undefined
 
+export interface VariableData {
+  // name: string;
+  // type: VariableType;
+  // data: {
```

Comment:
Should these commented out properties be removed?

Reply:
👍